src/plugins/Makefile.in

```diff
@@ -33,7 +33,7 @@
 #                                                                    #
 # Note: DO NOT include the .c extension!!!                           #
 
-MYPLUGINS = 
+#MYPLUGINS = 
```

Comment:
better replace to ``MYPLUGINS := $(MYPLUGINS)``
in this way will works old way and you can set env var too